src/main/java/com/denizenscript/depenizen/bukkit/bridges/JobsBridge.java

```diff
@@ -1,70 +1,86 @@
-        PropertyParser.registerProperty(JobPlayer.class, JobsJobTag.class);
+        /////////////////
+        // Register Events
+        /////////////////
+        ScriptEvent.registerScriptEvent(JobsJobsPaymentScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsExpGainScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsJoinScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsLeaveScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsLevelUpScriptEvent.class);
+        /////////////////
+        // Register Tag and Properties
+        /////////////////
+        ObjectFetcher.registerWithObjectFetcher(JobsJobTag.class, JobsJobTag.tagProcessor);
         PropertyParser.registerProperty(JobsPlayerProperties.class, PlayerTag.class);
-        TagManager.registerTagHandler(new TagRunnable.RootForm() {
-            @Override
-            public void run(ReplaceableTagEvent event) {
-                tagEvent(event);
-            }
-        }, "jobs");
-        DenizenCore.commandRegistry.registerCommand(JobsCommand.class);
-    }
+        PropertyParser.registerProperty(JobsJobProperties.class, JobsJobTag.class);
 
-    public void tagEvent(ReplaceableTagEvent event) {
-        Attribute attribute = event.getAttributes();
-        JobsJobTag j = null;
-        if (attribute.hasParam()) {
-            // Documented below.
-            if (JobsJobTag.matches(attribute.getParam())) {
-                j = attribute.paramAsType(JobsJobTag.class);
+        /////////////////
+        // Register root tag handler
+        /////////////////
+
+        // <--[tag]
+        // @attribute <jobs[<name>]>
+        // @returns JobsJobTag
+        // @plugin Depenizen, Jobs
+        // @description
+        // Returns the job tag with the given name
+        // -->
+        TagManager.registerTagHandler(ObjectTag.class, "jobs", (attribute) -> {
+            if (attribute.hasParam()) {
+                return JobsJobTag.valueOf(attribute.getParam(), attribute.context);
             }
-            else {
-                attribute.echoError("Could not match '" + attribute.getParam() + "' to a valid job!");
-                return;
+            // <--[tag]
+            // @attribute <jobs.server_jobs>
+            // @returns ListTag(JobsJobTag)
+            // @plugin Depenizen, Jobs
+            // @description
+            // Returns the list of all jobs on the server
+            // -->
+            if (attribute.startsWith("server_jobs", 2)) {
+                attribute.fulfill(1);
+                ListTag jobsList = new ListTag();
+                for (Job job : Jobs.getJobs()) {
+                    jobsList.addObject(new JobsJobTag(job));
+                }
+                return jobsList;
             }
-        }
-        else {
             // <--[tag]
-            // @attribute <jobs[(<name>)]>
-            // @returns ListTag(JobsJobTag)/JobsJobTag
+            // @attribute <jobs>
```

Comment:
`<jobs>` and `<jobs[<name>]>` is the same meta entry twice, with conflicting data. This can't work.